### PR TITLE
Create an adapter for persisting requests to Dependency Monkey

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/adviser/pull/1670

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thoth/storages/dependency_monkey.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Adapter for persisting requests to Dependency Monkey runs."""
+
+from .result_base import ResultStorageBase
+
+
+class DependencyMonkeyStore(ResultStorageBase):
```
Comment (pacospace) [inline thoth/storages/dependency_monkey.py:23]: what about `DependencyMonkeyRequestsStore`? to be consistent with `DependencyMonkeyReportsStore`: https://github.com/thoth-station/storages/blob/master/thoth/storages/dependency_monkey_reports.py

Comment (fridex) [reply thoth/storages/dependency_monkey.py]: Adjusted in 43e61fa :+1:

**Review by pacospace — APPROVED** [review]
Thanks ☕

**Pull request merged**